Added "Find ESTP System"; Removed "Early Adopter"

julieplummer20 opened the pull request.

MichaelCzcz commented on tutorials/abap-s4hanacloud-procurement-purchasereq-checks/abap-s4hanacloud-procurement-purchasereq-checks.md:
```diff
@@ -1,31 +1,35 @@
+
+
+## Intro
+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?loc ...
```
Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21

MichaelCzcz commented on tutorials/abap-s4hanacloud-procurement-purchasereq-fldctrl/abap-s4hanacloud-procurement-purchasereq-fldctrl.md:
```diff
@@ -1,26 +1,32 @@
-### Overview: Defining and implementing the enhancement
+
+## Intro
+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?loc ...
```
Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21

MichaelCzcz commented on tutorials/abap-s4hcloud-procurement-po-customlogic-tracing/abap-s4hcloud-procurement-po-customlogic-tracing.md:
```diff
@@ -1,86 +1,89 @@
 - How to check the custom logic of this custom field, in **Custom Fields**
 
+## Intro
+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?loc ...
```
Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21

MichaelCzcz reviewed (changes requested):
I think that if a note is the only text in the Intro section, it does not have to be formatted as a note.

julieplummer20 commented:
Hi,

“I think that if a note is the only text in the Intro section, it does not have to be formatted as a note.”

It isn’t 😉:

***@***.***

I think it should be separate because the rest of the intro is specific to this tutorial “Implement a BAdI”, whereas the Note is generic to all SAP ABAP Cloud Environment tutorials.

However, I have removed the word “NOTE”.

BR Julie.

From: Michael Czadankiewicz ***@***.***>
Sent: Tuesday, 22 November 2022 12:47
To: sap-tutorials/Tutorials ***@***.***>
Cc: Plummer, Julie ***@***.***>; Author ***@***.***>
Subject: Re: [sap-tutorials/Tutorials] Added "Find ESTP System"; Removed "Early Adopter" (PR #20737)


@MichaelCzcz requested changes on this pull request.

I think that if a note is the only text in the Intro section, it does not have to be formatted as a note.

________________________________

In tutorials/abap-s4hanacloud-procurement-purchasereq-checks/abap-s4hanacloud-procurement-purchasereq-checks.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F20737%23discussion_r1029227409&data=05%7C01%7Cjulie.plummer%40sap.com%7Caebfd20272154de3605a08dacc7f535e%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638047144479711341%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=X8WS%2Bh%2BHfk0Ij4%2Bc4EM%2FZ5YiCWyBUZ9tNvHK4hKl4fc%3D&reserved=0>:

>  - How to log on to SAP S/4HANA Cloud ABAP Environment

 - How to create an ABAP package

 - How to find relevant existing `BAdI` enhancement spots for your line of business, in this case **Materials-Management-Purchasing**

-- How to implement checks for the field quantity and delivery date during creation of purchase requisition item

+- How to implement a `BAdI` that checks for the field quantity and delivery date during creation of purchase requisition item

+

+

+

+## Intro

+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?locale=en-US).

Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fjam4.sapjam.com%2Fwiki%2Fshow%2FMozDWaPczznL8NKeGEk0y9%3F_lightbox%3Dtrue%23mcetoc_1dtqfqchq21&data=05%7C01%7Cjulie.plummer%40sap.com%7Caebfd20272154de3605a08dacc7f535e%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638047144479711341%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=fkNcYsMikQDIZrW%2BE1haRbAQHU7%2BdtOJh174TMltlGk%3D&reserved=0>

________________________________

In tutorials/abap-s4hanacloud-procurement-purchasereq-fldctrl/abap-s4hanacloud-procurement-purchasereq-fldctrl.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F20737%23discussion_r1029228387&data=05%7C01%7Cjulie.plummer%40sap.com%7Caebfd20272154de3605a08dacc7f535e%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638047144479867565%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=tVFBi3HHuf0dWN9xltLjrdeblMFljrErXGIDRFQ%2F03k%3D&reserved=0>:

>  - How to logon to SAP S/4HANA Cloud ABAP Environment

 - How to create an ABAP package

 - How to find relevant existing `BAdI` enhancement spots for your line of business, in this case **Materials-Management-Purchasing**

 - How to hide a field, **Price**, by changing the status of a field control



-### Overview: Defining and implementing the enhancement

+

+## Intro

+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?locale=en-US).

Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fjam4.sapjam.com%2Fwiki%2Fshow%2FMozDWaPczznL8NKeGEk0y9%3F_lightbox%3Dtrue%23mcetoc_1dtqfqchq21&data=05%7C01%7Cjulie.plummer%40sap.com%7Caebfd20272154de3605a08dacc7f535e%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638047144479867565%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=geRn34q%2BFKkOCz73DA6neirV%2FFVjZ6wm74dtV%2BSJG9g%3D&reserved=0>

________________________________

In tutorials/abap-s4hcloud-procurement-po-customlogic-tracing/abap-s4hcloud-procurement-po-customlogic-tracing.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F20737%23discussion_r1029229153&data=05%7C01%7Cjulie.plummer%40sap.com%7Caebfd20272154de3605a08dacc7f535e%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638047144479867565%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kubsz5kmLFKYxUf8ELuhyNH1PDLzmBDq0Q8AadaWlgw%3D&reserved=0>:

>  - How to create custom logic for an existing custom field in the app **Custom Fields**

 - How to run a trace for the Fiori app **Create Purchase Order** in the app **Custom Logic Tracing**

 - How to check the output value of this custom field using **Custom Logic Tracing**

 - How to check the custom logic of this custom field, in **Custom Fields**



+## Intro

+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?locale=en-US).

Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fjam4.sapjam.com%2Fwiki%2Fshow%2FMozDWaPczznL8NKeGEk0y9%3F_lightbox%3Dtrue%23mcetoc_1dtqfqchq21&data=05%7C01%7Cjulie.plummer%40sap.com%7Caebfd20272154de3605a08dacc7f535e%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638047144479867565%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=geRn34q%2BFKkOCz73DA6neirV%2FFVjZ6wm74dtV%2BSJG9g%3D&reserved=0>

—
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F20737%23pullrequestreview-1189895986&data=05%7C01%7Cjulie.plummer%40sap.com%7Caebfd20272154de3605a08dacc7f535e%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638047144479867565%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=b40L6%2BxRDZBtYO1e%2FMX65l7%2FIOnYHjBvxW7ABXal6Vg%3D&reserved=0>, or unsubscribe<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFNPXTINE6LTI35OYCYN333WJSXEZANCNFSM6AAAAAASHT4BAE&data=05%7C01%7Cjulie.plummer%40sap.com%7Caebfd20272154de3605a08dacc7f535e%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638047144479867565%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=9FErfvN0CJMZCn9cFGYZslRRlCDobdilDpLnxS1PW94%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

MichaelCzcz commented on tutorials/abap-s4hanacloud-procurement-purchasereq-fldctrl/abap-s4hanacloud-procurement-purchasereq-fldctrl.md:
```diff
@@ -251,6 +260,7 @@ Check that yours is the implementation that will be called:
 ### More Information
```
Please use sentence case.

MichaelCzcz commented on tutorials/abap-s4hanacloud-procurement-purchasereq-checks/abap-s4hanacloud-procurement-purchasereq-checks.md:
```diff
@@ -288,7 +297,7 @@ Your new order appears in the Overview List.
 ### More Information
```
Please use sentence case.

MichaelCzcz commented on tutorials/abap-environment-external-api/abap-environment-external-api.md:
```diff
@@ -8,156 +10,244 @@ author_name: Julie Plummer
 ### Test yourself
 
 
+### More Information
```
Sentence case please :-)

MichaelCzcz commented on tutorials/abap-environment-external-api/abap-environment-external-api.md:
```diff
@@ -8,156 +10,244 @@ author_name: Julie Plummer
+3. Format (`Ctrl+F1`), save (`Ctrl+S`) and activate (`Ctrl+F3`) your class.
+
+
+### Run class in ABAP Console
```
Shouldn't it be "console" with a lower case C or is ABAP Console a product name?

MichaelCzcz commented on tutorials/abap-environment-external-api-json/abap-environment-external-api-json.md:
```diff
@@ -0,0 +1,339 @@
+
+To be able to access the external service, you must:
+
+- create an HTTP client that points to your communication scenario and service id
```
Shouldn't this an the other line items start with upper case?

MichaelCzcz reviewed (changes requested):
Please fix the image links! Border and link should be in one line

julieplummer20 commented:
Hi,

Discussed with Merve.

This is no longer as urgent, so I have sent only to QA anyway.

So next week is now fine.

“Shouldn't it be "console" with a lower case C or is ABAP Console a product name?”
It is Console (upper case) in the official docu. cf “SQL Console”.


“Shouldn't this an the other line items start with upper case?”

Not sure . Have changed to upper case.



Thanks for all your help.

Best wishes,

Julie.



From: Michael Czadankiewicz ***@***.***>
Sent: Thursday, 24 November 2022 13:29
To: sap-tutorials/Tutorials ***@***.***>
Cc: Plummer, Julie ***@***.***>; Author ***@***.***>
Subject: Re: [sap-tutorials/Tutorials] Added "Find ESTP System"; Removed "Early Adopter" (PR #20737)


@MichaelCzcz requested changes on this pull request.

Please fix the image links! Border and link should be in one line

________________________________

In tutorials/abap-environment-external-api/abap-environment-external-api.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F20737%23discussion_r1031453165&data=05%7C01%7Cjulie.plummer%40sap.com%7C1706ae12f401425b44be08dace1773e4%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638048897370027046%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=iPgYRCxubfRUVakxttB47n5CKxdcsPn1KziP114nM2w%3D&reserved=0>:

>



 ### Test yourself





+### More Information

Sentence case please :-)

________________________________

In tutorials/abap-environment-external-api/abap-environment-external-api.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F20737%23discussion_r1031453764&data=05%7C01%7Cjulie.plummer%40sap.com%7C1706ae12f401425b44be08dace1773e4%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638048897370183262%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=9Ax5uBwHP4NyIbY4BOgO3a4R5wwCI3uQ7xAG6PQOfz0%3D&reserved=0>:

> +

+          " catch any error: In case XSLT is not correct

+              CATCH cx_root INTO DATA(lx_root).

+                out->write( lx_root->get_text( ) ).

+          ENDTRY.

+

+      ENDMETHOD.

+

+    ```

+

+2. Check your syntax (`Ctrl+F2`).

+

+3. Format (`Ctrl+F1`), save (`Ctrl+S`) and activate (`Ctrl+F3`) your class.

+

+

+### Run class in ABAP Console

Shouldn't it be "console" with a lower case C or is ABAP Console a product name?

________________________________

In tutorials/abap-environment-external-api-json/abap-environment-external-api-json.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F20737%23discussion_r1031455997&data=05%7C01%7Cjulie.plummer%40sap.com%7C1706ae12f401425b44be08dace1773e4%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638048897370183262%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=bfiFrxDBFLhzsVSXHxYEKZDn6AlybQ0bPuMeOVOqTEs%3D&reserved=0>:

> +    <!-- border -->

+    ![step4e-class-plus-interface](step4e-class-plus-interface.png)

+

+4. Choose or create a transport request, then choose **Finish**.

+

+The class is displayed in a new editor.

+

+<!-- border -->

+![step4d-class-editor](step4d-class-editor.png)

+

+### Copy code

+

+

+To be able to access the external service, you must:

+

+- create an HTTP client that points to your communication scenario and service id

Shouldn't this an the other line items start with upper case?

—
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F20737%23pullrequestreview-1193073502&data=05%7C01%7Cjulie.plummer%40sap.com%7C1706ae12f401425b44be08dace1773e4%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638048897370183262%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=MwhqfbR2W5i0kFD%2FQS19mL1kP3HVScYwMWnCSsuSXZQ%3D&reserved=0>, or unsubscribe<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFNPXTMMGC5GJPGUEOJJQM3WJ5NQNANCNFSM6AAAAAASHT4BAE&data=05%7C01%7Cjulie.plummer%40sap.com%7C1706ae12f401425b44be08dace1773e4%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C638048897370183262%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=S2P3plAQOv5Q3B2ASoyK2ki4z%2FSeWLt3L49W7xSA8Mo%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>
